docs/user/running-cincinnati.md

```diff
@@ -16,6 +16,34 @@ The Cincinnati deploymentconfig is available [here](../../dist/openshift/cincinn
 oc new-project cincinnati
 ```
 
+#### Create secret for container registry
+
+If Cincinnati needs credentials to access a container registry to fetch the release payload information , you need to create a secret with name `cincinnati-credentials`. Make sure the auth for the container registry is present in the config.json file.
```

Comment:
```suggestion
If Cincinnati is configured to use services which require authentication, you need to create a secret with name `cincinnati-credentials`. Make sure the auth for the container registry is present in the config.json file to access a container registry with release payloads.
```

`cincinnati-credentials` secret may contain other credentials

Reply:
Fixed in https://github.com/openshift/cincinnati/pull/296/commits/57b9b01ed74d0a80299ea81a2d01047f1b6b6bc8